frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -452,6 +457,31 @@ pub struct ReadySolution<A> {
 	pub compute: ElectionCompute,
 }
 
+/// Try to convert the supports of a solution into a [`ReadySolution`].
+///
+/// Errors if any of the supports has more than [`MaxBackersPerWinner`] backers.
+impl<AccountId: PartialEq, MaxBackersPerWinner: Get<u32>>
```

Comment:
Instead of this, I would implement just `impl TryFrom<Supports<AccountId>> for BoundedSupports<..>` in `election-provider-support/src`.

The other two elements of the tuple seems very out of context.

Reply:
Regardless, it also really sucks that we have to re-allocate the buffer here. In my previous work, I looked into it and it wasn't possible within safe rust, but I am happy to break the taboo here and just use unsafe rust.

Reply:
Also, IIRC a lot of such conversions need to re-allocate in safe rust but with unsafe we can circumvent it.